Makefile

```diff
@@ -40,6 +41,13 @@ vm/parser.go: vm/parser.y .gen-dep-stamp
 emgen/emgen: emgen/emgen.go
 	cd emgen && go build
 
+install_gox:
+	go get github.com/mitchellh/gox
+
+crossbuild: install_gox $(GOFILES) .dep-stamp
```

Comment:
Insert surprised face emoji here.  Wow.  I didn't realise it was this easy!  Thanks.